elasticsearch/src/main/java/com/tcdi/zombodb/highlight/AnalyzedField.java

```diff
@@ -5,9 +5,9 @@
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
  * You may obtain a copy of the License at
- *
+ * 
```

Comment:
I've noticed this in a number of the files you've changed.  What are you doing that causes a space to be added after these lines?  Is it an editor/IDE thing you can turn off?  Whitespace changes are just noise in a large diff.